src/test-run/index.js

```diff
@@ -249,3 +278,22 @@ ServiceMessages[CLIENT_MESSAGES.ready] = function (msg) {
 
     return new Promise((resolve, reject) => this.pendingRequest = { resolve, reject });
 };
+
+ServiceMessages[CLIENT_MESSAGES.executePendingBrowserWindowCommand] = async function (msg) {
+    this.debugLog.driverMessage(msg);
+
+    var command        = this.pendingBrowserWindowCommand;
+    var executionError = null;
+
+    try {
+        if (command.type === COMMAND_TYPE.resizeWindow)
+            await resizeWindow(msg.pageUrl, command.width, command.height);
+        else if (command.type === COMMAND_TYPE.resizeWindowToFitDevice)
+            await resizeWindow(msg.pageUrl, command.device, command.orientation);
+    }
+    catch (err) {
+        executionError = new ActionUnsupportedDeviceTypeError();
```

Comment:
May be we should delegate error creation to browserNatives.resizeWindow method and return just err (not ActionUnsupportedDeviceTypeError), because it can be another error instead of this?
So in this case we should create error message like this: `Browser natives returns error:....`
What do you think @inikulin @VasilyStrelyaev ?